cms/templates/settings.html

```diff
@@ -251,10 +253,45 @@ <h2 class="title-2">${_('Course Schedule')}</h2>
           % if can_show_certificate_available_date_field(context_course):
           <ol class="list-input">
             <li class="field-group field-group-certificate-available" id="certificate-available">
-              <div class="field date" id="field-certificate-available-date">
+              <div class="field date" id="field-certificates-display-behavior">
+                <label for="certificates-display-behavior">${_("Certificates Display Behavior")}</label>
```

Comment:
It looks like you are translating here, but not anywhere else in the new strings you added. Is that right?